### PR TITLE
ETQB, lorsque je me connecte sur mon CFN, et que je change la langue, je reçois les emails & SMS dans cette langue

### PR DESCRIPTION
**Pull request by AntoineAresu**
[Ticket](https://trello.com/c/ycRmCFH4/4376-3-enqu%C3%AAte-3h-etqb-lorsque-je-me-connecte-sur-mon-cfn-et-que-je-change-la-langue-je-re%C3%A7ois-les-emails-sms-dans-cette-langue)

### CODE REVIEW

**Review by tcheymol — APPROVED** [review]

File: src/Controller/SecurityController.php [hunk 1]
```diff
@@ -40,30 +44,17 @@ public function redirectUser(): ?RedirectResponse
...
-        if (!$user instanceof User || is_string($user)) {
+        try {
+            return match (true) {
+                !$user instanceof User || is_string($user) => $this->redirect($this->generateUrl('re_main_login')),
+                $user->isAdministrateur() => $this->redirect($this->generateUrl('sonata_admin_dashboard')),
+                $user->isFirstVisit() => $this->redirect($this->generateUrl('re_user_firstVisit')),
+                $user->isBeneficiaire() => $this->redirect($this->generateUrl('beneficiary_home')),
+                $user->isMembre() => $this->redirect($this->generateUrl($this->isGranted(BeneficiaryVoter::MANAGE) ? 'list_beneficiaries' : 'affiliate_beneficiary_home')),
+            };
+        } catch (\UnhandledMatchError) {
```
Comment (NilsRec) [inline src/Controller/SecurityController.php:55]: Why not using 'default' in match ?

File: src/ServiceV2/ResettingService.php [hunk 2]
```diff
@@ -108,7 +108,7 @@ public function isRequestingBySMS(?User $user): bool
         return 0 < count($requests) && null !== $requests[0]->getSmsCode();
     }
 
-    public function processSendingPasswordResetEmail(string $email, string $locale): void
+    public function processSendingPasswordResetEmail(string $email): void
```
Comment (NilsRec) [inline src/ServiceV2/ResettingService.php:111]: I'm not a fan of this method name. `sendPasswordResetEmail` would be better in my opinion. 
We should avoid using the word process in methods names as much as possible as it is way too generic and gives so little information about what the method is actually doing.

File: src/ServiceV2/ResettingService.php [hunk 3]
```diff
@@ -34,9 +34,9 @@ public function __construct(
     ) {
     }
 
-    public function handleEmailSend(User $user, ResetPasswordToken $token, string $locale): void
+    public function handleEmailSend(User $user, ResetPasswordToken $token): void
```
Comment (NilsRec) [inline src/ServiceV2/ResettingService.php:37]: Same here, `sendEmail` is simpler and clearer. "handle" as "process" is too generic and should be avoided if possible.

File: src/ServiceV2/ResettingService.php [hunk 4]
```diff
@@ -65,7 +65,7 @@ public function getSmsCode(User $user): ?string
      */
     public function handleSmsSend(User $user, string $smsCode): void
```
Comment (NilsRec) [inline src/ServiceV2/ResettingService.php:66]: `sendSms` ?

**Pull request merged**